### PR TITLE
Andres Cabrera

### PR DESCRIPTION
**Pull request by acabrera100**

### CODE REVIEW

File: HW_classes.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+//     this.age = age;
+//   }
+// }
+// Human.prototype.ageOneYear = function() {
+//   this.age++;
+//   return `${this.age}`;
+// };
+// Human.prototype.eating = function() {
+//   console.log("mmm, mmm, mmm, I'm love'n it");
+// };
```
Comment (michellelala) [inline HW_classes.js:13]: Really great job! Alternatively, you could have placed these functions within the class using the syntax: 
```
ageOneYear() {
  return this.age++
}
```
Same goes for your `eating` function!

File: HW_classes.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+let A = new Vector(1, 2);
+let B = new Vector(2, 3);
+let O = new Vector(0, 0);
+let C = new Vector(3, 4);
+
+console.log(A.plus(B));
+console.log(B.minus(O));
+
+console.log(C.getLength()); // => 5
```
Comment (michellelala) [inline HW_classes.js:49]: Nice 👍

File: HW_classes.js (hunk 1 above)
Comment (XavierC4Q) [inline HW_classes.js:13]: We are going to want to practice ES6 ways of writing these out but good job.

File: HW_classes.js (hunk 2 above)
Comment (XavierC4Q) [inline HW_classes.js:49]: Great job. Nice and clear.

**Pull request closed without merging**